parkhangs-webapp/src/components/GoogleMap.js

```diff
@@ -0,0 +1,63 @@
+import React, {Component} from 'react'
+import { Map, GoogleApiWrapper, Marker } from 'google-maps-react'
+import 'components/GoogleMap.css'
+
+class GoogleMap extends Component {
+    static defaultProps = {
+        zoom: 14
+    }
+
+    defaultLocation = {
+        lat: 49.242140,
+        lng: -123.112158,
+    }
+
+    mapStyles = {
+        width: '100%',
+        height: '100%'
+    }
+
+    render() {
+        const parks = this.props.parks
+        const image = 'https://raw.githubusercontent.com/cmaija/ParkHangs/master/parkhangs-webapp/src/assets/icons/marker.png'
```

Comment:
why is the source from there? shouldn't it be using the asset from the project?

Reply:
because the new package required the marker image to be from a url and this was the easiest way to do it https://www.npmjs.com/package/google-maps-react#marker